spec/factories/factories.rb

```diff
@@ -323,5 +323,72 @@
     version_num nil
     round nil
     is_submitted false
+	end
+
+  factory :response_take, class: Response do
+    response_map { ResponseMap.first || association(:response_map_review) }
+    additional_comment nil
+    version_num nil
+    round 1
+    is_submitted true
+  end
+  
+	factory :response_map, class: ResponseMap do
+    id 200 
+  end
+
+	factory :response_map_metareview, class: ResponseMap do
+		type 'MetareviewResponseMap'
+	end
+
+	factory :metareviewresponsemap, class: MetareviewResponseMap do
+		id 6
+		reviewee_id 1
+		reviewed_object_id 8
+  end
+
+	factory :response_map_review, class: ResponseMap do
+		reviewee_id 1
+		reviewer_id 1
+		type 'ReviewResponseMap'
+	end
+
+  factory :selfreviewresponsemap, class: SelfReviewResponseMap do
```

Comment:
The difference between different response_map is only `type` in DB. It will be better to remove `review_response_map` factory in current factories and create a factory named `response_map` and then assign different types when using it.